### ISSUE TITLE
Minimum image dimensions

### ISSUE THREAD

**Issue by cconover** [pressupinc/require-featured-image#1]
It would be great if the plugin could enforce minimum image dimensions.

**Comment by hubdotcom** [comment]
Potentially fire just on thumbnail updates?

```
add_filter('wp_handle_upload_prefilter','tc_handle_upload_prefilter');
function tc_handle_upload_prefilter($file)
{

    $img=getimagesize($file['tmp_name']);
    $minimum = array('width' => '640', 'height' => '480');
    $width= $img[0];
    $height =$img[1];

    if ($width < $minimum['width'] )
        return array("error"=>"Image dimensions are too small. Minimum width is {$minimum['width']}px. Uploaded image width is $width px");

    elseif ($height <  $minimum['height'])
        return array("error"=>"Image dimensions are too small. Minimum height is {$minimum['height']}px. Uploaded image height is $height px");
    else
        return $file; 
}
```

**Comment by davidbhayes** [comment]
Solved by #8. Thanks again @cjk508!
